src/hooks/useRedirect.tsx

```diff
@@ -0,0 +1,16 @@
+import { useHistory } from "react-router-dom";
+
+interface Redirect {
+    redirect: (route: string, e?: React.MouseEvent<HTMLButtonElement, MouseEvent> | React.FormEvent<HTMLFormElement>) => void;
+}
+
+export function useRedirect(): Redirect {
+    const history = useHistory();
+
+    const redirect = (route: string, e?: React.MouseEvent<HTMLButtonElement, MouseEvent> | React.FormEvent<HTMLFormElement>) => {
+        e?.preventDefault();
+        history.push(route);
```

Comment:
I'm not sure on this hook. The 'redirect' verb makes me think history.replace(route) rather than history.push(route). I think our use case is calling for history.push(route). Am I being pedantic, or can you see some possible confusion in intent of the function?